app/src/main/java/net/bible/service/db/bookmark/BookmarkDatabaseDefinition.java

```diff
@@ -114,7 +120,8 @@ private void bootstrapDB(SQLiteDatabase db) {
                 BookmarkColumn.VERSIFICATION + " TEXT," +
                 BookmarkColumn.CREATED_ON + " INTEGER DEFAULT 0," +
                 BookmarkColumn.PLAYBACK_SETTINGS + " TEXT DEFAULT NULL" +
```

Comment:
There's syntax error in SQL syntax, which caused tests to fail.

```suggestion
                 BookmarkColumn.PLAYBACK_SETTINGS + " TEXT DEFAULT NULL," +
```

But before making this fix, please make sure you can run tests correctly in your environment (so that they fail without this fix) :)